src/test/java/io/confluent/connect/jdbc/sink/PreparedStatementBinderTest.java

```diff
@@ -137,9 +140,16 @@ public void bindRecordInsert() throws SQLException, ParseException {
     verify(statement, times(1)).setDouble(index++, valueStruct.getFloat64("double"));
     verify(statement, times(1)).setBytes(index++, valueStruct.getBytes("bytes"));
     verify(statement, times(1)).setBigDecimal(index++, (BigDecimal) valueStruct.get("decimal"));
-    verify(statement, times(1)).setDate(index++, new java.sql.Date(((java.util.Date) valueStruct.get("date")).getTime()), DateTimeUtils.UTC_CALENDAR.get());
-    verify(statement, times(1)).setTime(index++, new java.sql.Time(((java.util.Date) valueStruct.get("time")).getTime()), DateTimeUtils.UTC_CALENDAR.get());
-    verify(statement, times(1)).setTimestamp(index++, new java.sql.Timestamp(((java.util.Date) valueStruct.get("timestamp")).getTime()), DateTimeUtils.UTC_CALENDAR.get());
+    Calendar utcCalendar = DateTimeUtils.getTimeZoneCalendar(TimeZone.getTimeZone(ZoneOffset.UTC));
+    verify(statement, times(1))
+        .setDate(index++, new java.sql.Date(((java.util.Date) valueStruct.get("date")).getTime()),
+            utcCalendar);
+    verify(statement, times(1))
+        .setTime(index++, new java.sql.Time(((java.util.Date) valueStruct.get("time")).getTime()),
+            utcCalendar);
+    verify(statement, times(1)).setTimestamp(index++,
+        new java.sql.Timestamp(((java.util.Date) valueStruct.get("timestamp")).getTime()),
+        utcCalendar);
```

Comment:
Nit: would be good to clean up the formatting of these methods to match our multi-line method call style, and check the changes in the other tests, too.